src/pages/Detail/index.tsx

```diff
@@ -84,6 +86,7 @@ export const DetailPage = () => {
   const basicPrice = price?.basicPrice ?? 0;
   const totalPrice = basicPrice * productCount;
   const priceString = `${basicPrice.toLocaleString()}원`;
+  const giftOrderLimit = productOption?.giftOrderLimit || 0;
```

Comment:
```suggestion
  const giftOrderLimit = useMemo(() => productOption?.giftOrderLimit || 0, [productOption?.giftOrderLimit]);
```
값이 계속 변경되서 그런걸수도 있어서, 이렇게 메모이제이션 해보세요!

Reply:
아 useMemo가 낯설었는데 이 기회에 더 공부해보겠습니당 감사합니다!!